test/tpu/Dockerfile

```diff
@@ -3,12 +3,12 @@ FROM us-central1-docker.pkg.dev/tpu-pytorch-releases/docker/development:tpu as b
 # Replace value with the latest runner release version
 # source: https://github.com/actions/runner/releases
 # ex: 2.303.0
-ARG RUNNER_VERSION="2.314.1"
+ARG RUNNER_VERSION="2.316.1"
```

Comment:
Do you know how to find the correct version of RUNNER_VERSION and RUNNER_CONTAINER_HOOKS_VERSION?